### ISSUE TITLE
#import "Appboy-iOS-SDK/AppboyKit.h" not found?

### ISSUE THREAD

**Issue by devshorts** [Appboy/appboy-ios-sdk#308]
# Report
I'm following the react native integration outlines here for podspec based installs

https://www.braze.com/docs/developer_guide/platform_integration_guides/react_native/ios/sdk_integration/

Which first links to the ios sdk integration link (note, the link linked to in the docs: https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/initial_sdk_setup/ is wrong...)

The ios sdk integration links says to add

```
#import "Appboy-iOS-SDK/AppboyKit.h"
```

But xcode just complains that it can't find it.  

After some digging I ended up doing 

```
#import <AppboyKit.h>
#import <AppboyReactUtils.h>
```
Which made the following two lines compile:

```
- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
....
  // Braze
  [Appboy startWithApiKey:@"YOUR-APP-IDENTIFIER-API-KEY"
          inApplication:application
          withLaunchOptions:launchOptions];
  [[AppboyReactUtils sharedInstance] populateInitialUrlFromLaunchOptions:launchOptions];
```

I'm not a native ios dev so I'm not sure if this is even right, but otherwise I'd need some guidance on what is the right import to pull in

## Describe your environment.

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS 
 Platform Version        | e.g. 12.1
 SDK Version             | e.g. 3.10.0
 Integration Method      | e.g. Cocoapods
 Xcode Version           | e.g. Xcode 12.5.1

```
  "name": "react-native-appboy-sdk",
  "version": "1.31.0",
```

**Comment by Bucimis** [comment]
@devshorts could you post your header search paths (`HEADER_SEARCH_PATHS`)? They should include a path to the Appboy pod's public headers

**Comment by devshorts** [comment]
@Bucimis  seems like  these two items were added automatically as part of the auto linking

![image](https://user-images.githubusercontent.com/1799346/127061091-3bdaaeb6-11df-493b-a414-ad89d9c61c4d.png) ("${PODS_CONFIGURATION_BUILD_DIR}/Appboy-iOS-SDK/Appboy_iOS_SDK.framework/Headers")

![image](https://user-images.githubusercontent.com/1799346/127061092-32436ef4-627a-4e11-b02e-c6baec526a51.png)

For what its worth, importing the two headers as mentioned in the original report did work, I was able to verify we are sending some sample data to the dashboard from our react native app

**Comment by Bucimis** [comment]
@devshorts do you have `!use_frameworks` in your `Podfile`? And yep, it should actually work either way - the import syntax will depend on how the SDK was ultimately compiled based on your configuration

**Comment by devshorts** [comment]
@Bucimis yes we have

```
use_frameworks! :linkage => :static
```

**Comment by Bucimis** [comment]
@devshorts 👍 that's almost certainly the reason, the SDK should function fine either way. we'll close this out for now, let us know if anything else comes up!

**Comment by ivannzme** [comment]
Run into the same issue, has this been resolved? I am on 1.37.0

**Comment by hokstuff** [comment]
Hi @ivannzme,

Version 1.37.0 of AppboyKit is a very old version, and AppboyKit itself has just passed [end-of-support as of March 1, 2024](https://github.com/Appboy/appboy-ios-sdk?tab=readme-ov-file#-the-new-braze-swift-sdk-is-now-available-), meaning we will no longer be developing on it nor fixing critical bugs on the repo. I encourage you to either see if the latest version of [AppboyKit 4.7.0](https://github.com/Appboy/appboy-ios-sdk/releases/tag/4.7.0) addresses your issues or to migrate to [the Swift SDK](https://github.com/braze-inc/braze-swift-sdk/), which will resolve the issue you are facing.

Thanks.
